src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
- [OPTIONAL] When using the modulus (%) with 0 it crashes the app. I think this should work in the same manner as dividing by 0 does. ♻️ 

![mod%0](https://user-images.githubusercontent.com/79876959/143510767-86c3f755-d949-4ad9-b93a-5181006ddcd2.PNG)